planning/behavior_velocity_no_stopping_area_module/src/scene_no_stopping_area.cpp

```diff
@@ -87,12 +91,8 @@ boost::optional<LineString2d> NoStoppingAreaModule::getStopLineGeometry2d(
         const LineString2d line{{p0.x, p0.y}, {p1.x, p1.y}};
         std::vector<Point2d> collision_points;
         bg::intersection(area_poly, line, collision_points);
-        if (collision_points.empty()) {
-          continue;
-        }
-        const double yaw = tier4_autoware_utils::calcAzimuthAngle(p0, p1);
```

Comment:
it is possible that `collision_points.empty()` , so this check is necessary

Reply:
same check seems to be done in later line

![image](https://github.com/autowarefoundation/autoware.universe/assets/39142679/1b263892-9bc8-411a-8a52-e8d2541ed8f6)